### PR TITLE
Sem-version change firefox version 52,68,78

### PR DESCRIPTION
**Pull request by bogyo210**

### CODE REVIEW

**Comment by shiroyasha** [comment]
👍 I like how the syntax `sem-version firefox 55` looks, it has a nice uniformity with the rest of the system.

What is the state of our documentation? When we mention `sem-version` are we super focused on programming languages? If yes, we would need to revisit that part before fully shipping this.

File: sem-version [hunk 1]
```diff
@@ -94,7 +94,18 @@ version::change_scala() {
...
+version::change_firefox() {
+  local language=""
+  case "${language_version}" in
+    "52")language="esr-prev";;
+    "68")language="esr";;
+    "78")language="esr-new";;
+    *)echo "Please choose one of 52, 68, 78";exit 1;;
+  esac
+  language_version="$language"
+  version::change "change-firefox-version"
```
Comment (shiroyasha) [inline sem-version:108]: The usage of the word `language` for handling non-language things makes this code hard to read and maintain. We need to improve this.

Comment (bogyo210) [reply sem-version:108]: Changed `language` to `ff_version`.

Comment (shiroyasha) [reply sem-version:108]: This one is still ambiguous `language_version="$ff_version"`. Let's refactor the script if necessary for long term clarity and maintainability.

**Comment by shiroyasha** [comment]
Looks great! Thanks for doing the extra refactoring work 🙇

**Pull request merged**